weave/index/index_pandas.py

```diff
@@ -424,6 +436,11 @@ def get_children(self, basket_address, **kwargs):
         data = data.drop_duplicates(
             subset=['uuid', 'generation_level']
         ).reset_index(drop=True)
+
+        # Keep all of the rows where the generation_level is more than
+        # or equal to the min_gen_level
+        data = data[data["generation_level"] >= min_gen_level]
```

Comment:
I think we want to change the recursion/queries so that we stop earlier, rather than filtering after the fact. That way we can save significant time in cases where there may be 100 generation levels, but we only want up until gen 5.